eessi_bot_job_manager.py

```diff
@@ -533,6 +561,33 @@ def process_finished_job(self, finished_job):
         comment_update = f"\n|{dt.strftime('%b %d %X %Z %Y')}|finished|"
         comment_update += f"{comment_description}|"
 
+        # check if _bot_jobJOBID.test exits
+        # TODO if not found, assume test was not run (or failed, or ...) and add
+        # a message noting that ('not tested' + 'test suite not run or failed')
+        # --> 'bot: test [FILTERs]' command runs tests when filter matches
```

Comment:
This comment is false, there is no `test` command implemented in the bot (yet), so this line should be removed.

We can/should instead mention that `bot/test.sh` and `bot/check-test.sh` scripts are run by job script run by bot for `build` command?

```suggestion
        # --> bot/test.sh and bot/check-test.sh scripts are run in job script used by bot for 'build' action
```